3-networks/README.md

```diff
@@ -40,6 +42,47 @@ The purpose of this step is to :
 1. Merge changes to nonprod with `git checkout -b nonprod` and `git push origin nonprod`
     1. Review the apply output in your cloud build project https://console.cloud.google.com/cloud-build/builds?project=YOUR_CLOUD_BUILD_PROJECT_ID
 
+### Setup to run via Jenkins
+1. Clone the repo you created manually in bootstrap: `git clone <YOUR_NEW_REPO-3-networks>`
+1. Navigate into the repo `cd YOUR_NEW_REPO_CLONE-3-networks` and change to a non prod branch `git checkout -b plan` (the branch `plan` is not a special one. Any branch which name is different from `dev`, `nonprod` or `prod` will trigger a terraform plan).
+1. Copy contents of foundation to new repo `cp -R ../terraform-example-foundation/3-networks/* .` (modify accordingly based on your current directory).
+1. Copy the `Jenkinsfile` to the root of your new repository (modify accordingly based on your current directory):
+   ```
+   cp ../terraform-example-foundation/build/Jenkinsfile .
+   ```
+1. Update the variables located in the `environment {}` section of the `Jenkinsfile` with values from your environment:
+    ```
+    _POLICY_REPO (optional)
+    _TF_SA_EMAIL
+    _STATE_BUCKET_NAME
+    ```
+1. Copy the `tf-wrapper.sh` configuration file to the root of your new repository (modify accordingly based on your current directory):
+    ```
+    cp ../terraform-example-foundation/build/tf-wrapper.sh .
+    ```
+1. Ensure wrapper script can be executed `chmod 755 ./tf-wrapper.sh`.
+1. Rename `common.auto.example.tfvars` to `common.auto.tfvars` and update the file with values from your environment and bootstrap.
+1. Rename `dev_nonprod_prod.auto.example.tfvars` to `dev_nonprod_prod.auto.tfvars` and update the file with the `access_context_manager_policy_id` value.
```

Comment:
@amandakarina and @atos-cit - FYI

Same here as in lines 25 and 26 -
These steps need clarification, because the files are not called like that - Amanda is working on a fix - https://github.com/terraform-google-modules/terraform-example-foundation/issues/169#issuecomment-667076579

Reply:
Got the most recent changes from Amanda here as well. Thank you.